src/constants/basePath.ts

```diff
@@ -0,0 +1,4 @@
+import * as config from '@/../next.config'
```

Comment:
I don't understand why this file is necessary?

If we have next.config.js

Reply:
It's just for shorting down the variable name. From config.basePath to basePath